src/views/product-tutorials-view/index.tsx

```diff
@@ -23,11 +24,16 @@ function ProductTutorialsView({
 
   const PageHeading = () => {
     const { title, level, slug } = getOverviewHeading()
-    const HeadingElem = `h${level}` as React.ElementType
     return (
-      <HeadingElem id={slug} className={s.heading}>
+      <Heading
+        id={slug}
+        level={level}
+        size={500}
+        weight="bold"
+        className={s.heading}
+      >
         {title}
-      </HeadingElem>
+      </Heading>
```

Comment:
It's hard cause I think we need an `<h1>` for the page - or at least, it felt like an okay way to meet some perceived requirements:
- I think we need an `<h1>` somewhere on the page, since it seems logical that we'd have multiple `<h2>`s - one for each section... but there's no visible page heading in designs 🤔 
- There's an `Overview` item in the table of contents that needs something to link to (this could be anything I suppose, not just an `<h1>`, but for some reason my mind went to headings since I feel like that's where anchor links usually link)

very open to other approaches / rethinking the requirements here 🙇‍♂️